content/documentation/staging/installation-guide/creating-updating-kiali-cr.adoc

```diff
@@ -0,0 +1,102 @@
+---
+title: "Creating and updating the Kiali CR"
+date: 2021-09-12T00:00:00+00:00
+draft: false
+weight: 40
+---
+
+:toc: macro
+:toclevels: 4
+:toc-title: In this section:
+:icons: font
+:sectlinks:
+:linkattrs:
+
+toc::[]
+
+The Kiali Operator watches the _Kiali Custom Resource_ (Kiali CR), a YAML file
+that holds the deployment configuration. Creating, updating, or removing a
+Kiali CR will trigger the Kiali Operator to install, update, or remove Kiali.
+
+The Operator provides comprehensive defaults for all properties of the Kiali
+CR. Hence, the minimal Kiali CR does not have a `spec`:
+
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+----
+
+Assuming you saved the previous YAML to a file named `my-kiali-cr.yaml`, you
+create the resource in the `istio-system` namespace with the following command:
+
+[source,bash]
+----
+$ kubectl apply -f my-kiali-cr.yaml -n istio-system
+----
+
+TIP: Typically, the Kiali CR is created in the Istio namespace. Thus, also Kiali is installed in this namespace.
+
+Once created, the Kiali Operator will process the resource and perform Kiali
+installation. You can check installation progress by inspecting the `status` attribute of the created Kiali CR:
+
+[source,bash]
+----
+$ kubectl describe kiali -n istio-system
+Name:         kiali
+Namespace:    istio-system
+Labels:       <none>
+Annotations:  <none>
+API Version:  kiali.io/v1alpha1
+Kind:         Kiali
+
+  (...some output is removed...)
+
+Status:
+  Conditions:
+    Last Transition Time:  2021-09-15T17:17:40Z
+    Message:               Running reconciliation
+    Reason:                Running
+    Status:                True
+    Type:                  Running
+  Deployment:
+    Instance Name:  kiali
+    Namespace:      istio-system
+  Environment:
+    Is Kubernetes:       true
+    Kubernetes Version:  1.21.2
+    Operator Version:    v1.40.0
+  Progress:
+    Duration:  0:00:16
+    Message:   5. Creating core resources
+Events:        <none>
+----
+
+IMPORTANT: All resources created by the Kiali Operator are managed by the
+Operator and should *not* be manually edited.
+
+We recommended to download the
+link:https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml[example
+Kiali CR YAML file] that is available in the Operator's GitHub repository. This
+example file contains and describes all available settings. Then, edit the
+downloaded file being careful to maintain proper formatting.
+
+NOTE: The link in the previous paragraph is for the example Kiali CR hosted in
+GitHub, in the `master` branch. Use GitHub's branch selector to see the example
+file that matches the Kiali Operator version that you are using.
```

Comment:
This is what we need to get away from, and get the example CR into kiali.io.

Reply:
Yes, I know... I think we need to keep this until everything is on kiali.io.